repose-aggregator/artifacts/valve/src/main/scala/org/openrepose/valve/ReposeJettyServer.scala

```diff
@@ -128,9 +128,7 @@ class ReposeJettyServer(val clusterId: String,
           }
         }
 
-        Option(ssl.isTlsRenegotiationAllowed).foreach { tls =>
-          cf.setRenegotiationAllowed(tls)
-        }
+        cf.setRenegotiationAllowed(Option(ssl.isTlsRenegotiationAllowed).getOrElse[java.lang.Boolean](java.lang.Boolean.FALSE))
```

Comment:
Using java.lang.Boolean.FALSE, you should be able to drop the explicit type on `getOrElse`.